src/charts/tree/tree.js

```diff
@@ -18,7 +18,7 @@ const defaultOptions = {
         collapsed: 'lightsteelblue',
         parent: 'white'
       },
-      radius: 5
+      radius: 7
```

Comment:
The radius from defaultOptions was not used at all, see `nodeUpdate.select('circle').attr('r', 7)` on line 245